docs/Makefile

```
@@ -3,15 +3,25 @@ DATADIR := ${PREFIX}/share
+export GITIGNORE := $(shell tail -n +2 .gitignore)
 
-docs:  $(patsubst %.md,%,$(wildcard *[157].md))
+MANPAGES_MD_IN := $(wildcard *.md.in)
+MANPAGES_MD_GENERATED := $(MANPAGES_MD_IN:%.md.in=%.md)
+MANPAGES_STAMP := .manpages-md.stamp
+
+# Wildcard alone misses generated *.md not yet present at make parse time.
+ALL_MD_SOURCES := $(sort $(MANPAGES_MD_GENERATED) $(wildcard *[157].md))
+DOCS_MAN_TARGETS := $(patsubst %.md,%,$(ALL_MD_SOURCES))
+
+docs: manpages-md $(DOCS_MAN_TARGETS)
```

checkmake (0.2.2)

[warning] Target "docs" should be declared PHONY.

(phonydeclared)